File: docs/examples/sdk/quickstart.py
from ailang_parse import DocParse

client = DocParse(api_key="dp_your_key_here")

result = client.parse("report.docx")
for block in result.blocks:
    if block.type == "heading":
        print(f"H{block.level}: {block.text}")
    elif block.type == "table":
        print(f"Table: {len(block.rows)} rows")
    elif block.type == "change":
        print(f"{block.change_type} by {block.author}")

# Unstructured migration — one import change:
from ailang_parse import UnstructuredClient
client = UnstructuredClient(server_url="https://docparse.ailang.sunholo.com")
elements = client.general.partition(file="report.docx")

File: docs/examples/sdk/quickstart.js
import { DocParse } from '@ailang/parse';

const client = new DocParse({ apiKey: 'dp_your_key_here' });

const result = await client.parse('report.docx');
for (const block of result.blocks) {
  if (block.type === 'heading') console.log(`H${block.level}: ${block.text}`);
  if (block.type === 'table') console.log(`Table: ${block.rows?.length} rows`);
  if (block.type === 'change') console.log(`${block.changeType} by ${block.author}`);
}

// Unstructured migration — one import change:
import { UnstructuredClient } from '@ailang/parse';
const uc = new UnstructuredClient({ serverUrl: 'https://docparse.ailang.sunholo.com' });
const elements = await uc.general.partition({ file: 'report.docx' });